rfc/trace_identifiers.md

```diff
@@ -0,0 +1,89 @@
+# Trace Identifiers
+
+**Current State:** Draft 
+**Author:** [tedsuo](https://github.com/tedsuo)
+
+The OpenTracing SpanContext interface is extended to include `SpanID` and `TraceID` accessors. 
+
+The OpenTracing model of computation specifies two primary object types, `Spans` and `Traces`, but does not specify identifiers for these objects. Identifiers for the two primary object types make it easier to correlate tracing data with data in other systems, simplify important tasks, and allow the creation of reusable trace observers. Some use cases are detailed below.
+
+# Background: Existing Protocols
+Before discussing changes to the OpenTracing specification, it’s worth reviewing several popular wire protocols which contain these trace identifiers.
+
+## Trace-Context HTTP Headers
+[Trace-Context HTTP headers](https://github.com/w3c/distributed-tracing) are in the process of being standardized via the w3c. The tracing community has voiced strong support in implementing these headers for use in tracing interop.
+
+The `Trace-Parent` header contains the following fields: `version`, `trace-id`, `span-id`, and `trace-options`.
+
+| field | format | description |
+| :---  | :---   | :---    |
+| `trace-id` | 16-byte array | The ID of the whole trace forest. If all bytes are 0, the `Trace-Parent` may be ignored. |
+| `span-id`  | 8-byte array  | The ID of the caller span (parent). If all bytes are 0, the `Trace-Parent` may be ignored. |
+
+## B3 HTTP Headers
+The [B3 HTTP headers](https://github.com/openzipkin/b3-propagation) are widely adopted, mostly by Zipkin-like tracing systems. The B3 protocol includes `X-B3-TraceId` and `X-B3-SpanId` as required headers, which contain the `TraceId` and `SpanId` values, respectively.
+
+| field | format | description |
+| :---  | :---   | :---        |
+| `TraceId` | 64 or 128-bit | The ID of the trace. Every span in a trace shares this ID. |
```

Comment:
Be consistent in describing the length in bits vs bytes (in TraceContext, you wrote the length is 16 bytes) . Bits is more commonly used I feel.

Reply:
Ok, I normalized on bits, and included a note about character space (HEXDIG vs opaque).